MvvmCross/Platform/Platform/IoC/MvxSimpleIoCContainer.cs

```diff
@@ -6,14 +6,15 @@
 // Project Lead - Stuart Lodge, @slodge, me@slodge.com
 
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Reflection;
 using MvvmCross.Platform.Core;
-using MvvmCross.Platform.Exceptions;
 
 namespace MvvmCross.Platform.IoC
 {
+    /// <summary>
+    /// Singleton IoCContainer.
+    /// 
+    /// Delegates to the <see cref="MvxIoCContainer"/> implementation
+    /// </summary>
     public class MvxSimpleIoCContainer
```

Comment:
We should maybe rename this class to something like `MvxIoCProvider`? It isn't really an IoC Container anymore.

Reply:
Yes, that would be good. I hope there aren't a lot of dependencies on this class.

Reply:
Related: The `MvxPropertyInjectingIoCContainer` is currently extending `MvxSimpleIoCContainer` but the singleton capability of this inheritance is never used. I'll change the class to extend the new `MvxIoCContainer`.

Reply:
Yeah, I think you can safely remove it 👍

Reply:
👍  Renamed `MvxSimpleIoCContainer` and dropped `MvxPropertyInjectingIoCContainer`